### DISCUSSION TITLE
Welcome to AMII Discussions!

### DISCUSSION THREAD

**Discussion by Unthrottled** [ani-memes/AMII#52] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dragon-0609** [comment]
> comment below with an introduction of yourself and tell us about what you do with this community.

Hi, I'm Dragon-LV (my nickname). I like `Doki Theme` and `AMII`. I have PHPStorm, and on it I have those great plugins. 

This year I'm going to learn Kotlin to be helpful with dev of the plugins. I don't know, what I would do, but I'll try to help you! 😉

**Reply by Unthrottled** [reply]
![Hello](https://media.giphy.com/media/keTwQbbQwlNM2RNJsW/giphy.gif)

If you are looking for something to do, the `Previous Meme` action 

https://github.com/ani-memes/AMII/blob/12693b611fe57f88e0e8325cd93e51911edff8f2/src/main/kotlin/io/unthrottled/amii/actions/PreviousMemeAction.kt#L11

Doesn't replay the sound (if the meme has one), when the action is invoked. If you wanted to get your feet wet, that might be a good starting point. 

[The intellij plugin template](https://github.com/JetBrains/intellij-platform-plugin-template#predefined-rundebug-configurations) is a good resource to look at, I think they have a comprehensive guide here https://plugins.jetbrains.com/docs/intellij/gradle-build-system.html

**Comment by Dragon-0609** [comment]
Thank you for your attention. 🙋‍♂️

> If you are looking for something to do

At the moment, I'm busy with the development of `Yuki Theme`, so I'm not looking for smth to do. I planned to start learning Kotlin in June. In that time I'll ask you what I can do.
